Auth0/Authentication.swift

```diff
@@ -631,12 +631,8 @@ public extension Authentication {
         return self.signup(email: email, username: username, password: password, connection: connection, userMetadata: userMetadata, rootAttributes: rootAttributes)
     }
 
-    func signup(email: String, username: String? = nil, password: String, connection: String, userMetadata: [String: Any]? = nil) -> Request<DatabaseUser, AuthenticationError> {
```

Comment:
This overload is not necessary because there is one above that adds default values to all the optional params. This one was adding default values to all but one. But in Swift you can omit a parameter if it has a default value (as per the overload above). So this one is pointless.